projects/the-application/src/assets/i18n/main/bn.json

```diff
@@ -1,17 +1,17 @@
 {
-  "ADD_TO_HOME_SCREEN": "ADD_TO_HOME_SCREEN",
-  "CLOSE": "CLOSE",
-  "DESCRIPTION": "DESCRIPTION",
-  "FULL_STOP": "FULL_STOP",
-  "GAME": "GAME",
-  "HIGH_SCORES": "HIGH_SCORES",
-  "INSTALL": "INSTALL",
-  "INSTALL_MEMORY_GAME": "INSTALL_MEMORY_GAME",
-  "PLEASE_RELOAD": "PLEASE_RELOAD",
-  "PLEASE_FINISH_OR_RESET_GAME": "PLEASE_FINISH_OR_RESET_GAME",
-  "RECENT_SCORES": "RECENT_SCORES",
-  "REFRESH": "REFRESH",
-  "SETTINGS": "SETTINGS",
-  "TITLE": "TITLE",
-  "UPDATE_AVAILABLE": "UPDATE_AVAILABLE"
+  "ADD_TO_HOME_SCREEN": "হোম পর্দায় যোগ করুন",
+  "CLOSE": "বন্ধ",
+  "DESCRIPTION": "বর্ণনা",
+  "FULL_STOP": "ফুলস্টপ্",
```

Comment:
From what @haldarsumana told me there is a different symbol for `.` (period) in Bengali. Can you set the value of `FULL_STOP` to that symbol?